test/bmc/bmc_api_test.rb

```diff
@@ -384,7 +384,7 @@ def test_api_can_pass_wrong_data_type_and_get_415_error
     Proxy::BMC::IPMI.any_instance.stubs(:bootbios).returns(true)
     put "/#{@host}/chassis/config/bootdevice/bios", "".to_json, "CONTENT_TYPE" => "application/json"
     assert_equal 415, last_response.status
-    assert_equal 'Invalid JSON content in body of request: data must be a hash, not String', last_response.body
+    assert last_response.body.start_with?('Invalid JSON content in body of request')
```

Comment:
JSON gem appears to return slightly different error messages depending on the version which was causing test failure on my machine.